### PR TITLE
fix(mantine): adjust Alert visual look

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Made some adjustments in the theme for the Alert component

- Icon should be 16px (was 20px)
- Space between icon and text should be 16px (was 24px)
- Padding around the alert should be 16px (was 24px)
- Content text should have color gray-7 (was gray-9)

| Before | After | Figma |
| --- | --- | --- |
| image | image | image |

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Comment by gdostie** [comment]
> Thank you for these adjustments 🙏 they're very appreciated. While we're at it, could we add a stroke to them, the lack of them have been a source of friction. If feasible, I would propose:
> 
> * Info = Navy1
> * Tip = Grey3
> * Success = Sec. Green1
> * Warning = Yellow2
> * Error = Red1
> 
> I've updated in Figma. If not feasible please let me know so I can remove them. Thanks again @gdostie !

@guscoveo Using the direct colors that you wanted for the border paused to be problematic since the shade used is different depending on the color. I managed to achieve a visually similar result by way of dark magic (using a [color-mix](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix) of the title color and the background color). Let me know if this is an acceptable compromise
image

**Comment by gdostie** [comment]
@guscoveo If you are wondering, I am using a mix of 15% title color, 85% background color:

```css
border-color: color-mix(in srgb, var(--alert-color), var(--alert-bg) 85%);
```

**Comment by guscoveo** [comment]
> @guscoveo If you are wondering, I am using a mix of 15% title color, 85% background color:
> 
> ```css
> border-color: color-mix(in srgb, var(--alert-color), var(--alert-bg) 85%);
> ```

I think this is an interesting compromise. That said it will be the case in the future that as we evolve on our tokens, actual tokens will be used for this kind of value, so something to keep in mind.

Thanks again @gdostie

**Review by guscoveo — APPROVED** [review]

File: packages/website/src/building-blocs/PageHeader.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export interface PageHeaderProps {
     title: string;
     thumbnail?: TileProps['thumbnail'];
     description?: ReactNode;
-    section: 'Foundations' | 'Layout' | 'Form' | 'Navigation' | 'Feedback' | 'Advanced' | 'Mantine';
+    section: 'Foundations' | 'Layout' | 'Form' | 'Navigation' | 'Feedback' | 'Advanced' | 'Mantine' | 'Feedbacl';
```
Comment (GermainBergeron) [inline packages/website/src/building-blocs/PageHeader.tsx:11]: There's a typo in `Feedbacl` and `Feedback` is already there
```suggestion
    section: 'Foundations' | 'Layout' | 'Form' | 'Navigation' | 'Feedback' | 'Advanced' | 'Mantine';
```

File: packages/website/src/pages/feedback/Alert.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        thumbnail="placeholder"
+        description="Attract user attention with important static message."
+        demo={<AlertDemo center />}
+        examples={{
+            tip: <AlertTipDemo center title="Tip" />,
+            success: <AlertSuccessDemo center title="Success" />,
+            warning: <AlertWarningDemo center title="Warning" />,
+            critical: <AlertCriticalDemo center title="Critical" />,
+        }}
+        sourcePath="/packages/mantine/src/components/button/Button.tsx"
```
Comment (GermainBergeron) [inline packages/website/src/pages/feedback/Alert.tsx:24]: This is not the correct link

**Review by GermainBergeron — APPROVED** [review]

File: packages/website/src/pages/feedback/Alert.tsx (hunk 2 above)
Comment (gdostie) [reply packages/website/src/pages/feedback/Alert.tsx]: updated thanks

**Pull request merged**